docs/DEPENDENCIES.md

```diff
@@ -0,0 +1,122 @@
+# Dependencies for deploying harbor-operator-app
+## Pre-deployment dependencies
+### Operational requirements
+In order to deploy the harbor-operator-app in a management cluster (MC), you need to have the following available:
+1. S3 Buckets: 
+   Two buckets are required for harbor-operator-app to function properly: 
+     - One for storing the WAL backups(if you plan to backup the postgres db) and 
+     - the other for storing harbor artifacts (Images, Charts etc.)
+   It is advised to use separate buckets for the WAL backups and the artifacts. Also, the bucket(s) must be accessible by the MC.
+3. AWS Credentials
+   The AWS credentials are needed for the harbor-operator to access the S3 bucket(s). The AWS entity (IAM User, IAM Role etc.) must have the following permissions:
+    - S3:ListBucket
+    - S3:GetObject
+    - S3:PutObject
+    - S3:CreateObject
+    - s3:GetBucket
+4. Subdomain
+    A subdomain is needed for the harbor-operator to create the ingress for the harbor instance. The subdomain must be accessible by the MC.
```

Comment:
Can we clarify this more? AFAIRC we just created an ingress which creates the subdomain

Reply:
I am fine with this PR once we have clarified this.

Reply:
I know that the harborcluster creates 2 ingress objects, `core` and `notary` (_based on our configuration_). As far as the subdomain is concerned, I'm not sure what creates that. Has someone manually created it, or do you have an operator running somewhere (something like `external-dns`)?